src/nni_manager/core/nniDataStore.ts

```diff
@@ -112,13 +114,18 @@ class NNIDataStore implements DataStore {
     }
 
     public async getTrialJob(trialJobId: string): Promise<TrialJobInfo> {
-        const trialJobs = await this.queryTrialJobs(undefined, trialJobId);
+        const trialJobs: TrialJobInfo[] = await this.queryTrialJobs(undefined, trialJobId);
 
         return trialJobs[0];
     }
 
     public async storeMetricData(trialJobId: string, data: string): Promise<void> {
-        const metrics = JSON.parse(data) as MetricData;
+        this.log.debug(`storeMetricData: trialJobId: ${trialJobId}, data: ${data}`);
+        const metrics: MetricData = JSON.parse(data);
+        if (metrics.type === 'REQUEST_PARAMETER') {
```

Comment:
Suggest to add comment to explain why we choose to skip to store 'REQUEST_PARAMETER' metric here

Reply:
Thanks, comments added.